README.adoc

```diff
@@ -271,20 +375,61 @@ system-deployment-694c7b74f7-lrlf7     0/1       Running   0          1m
 inventory-deployment-cf8f564c6-nctcr   1/1       Running   0          1m
 ----
 
-You will notice that one of the two `system` pods is no longer in the ready state. Navigate 
-to `{system-api}/properties`. Observe that your request will still be successful because you have two 
-replicas and one is still healthy.
+// Static guide instruction
+ifndef::cloud-hosted[]
+You will notice that one of the two `system` pods is no longer in the ready state.
+Navigate to `{system-api}/properties`.
+Observe that your request will still be successful because you have two replicas and one is still healthy.
```

Comment:
Observe that your request will still be successful because you have two replicas and one is still healthy.
--->
Your request is successful because you have two replicas and one is still healthy.